README.md

```diff
@@ -39,12 +39,42 @@ Binaries builds are turned on by default: `cargo build` will initiate a download
 from github to get the static lib. To disable this build using the
 `V8_FROM_SOURCE` environmental variable.
 
-If you prefer another location for downloading binaries,
-use the`RUSTY_V8_MIRROR` environmental variable.
-
 When making changes to rusty_v8 itself, it should be tested by build from
 source. The CI always builds from source.
 
+## The `RUSTY_V8_MIRROR` environment variable
+
+Tells the build script where to get binary builds from. Understands
+`http://`, `https://` and `file://` URLs, and defaults to
+https://github.com/denoland/rusty_v8/releases/download.
+
+`file://` mirrors are good for using cached downloads. First, point
+the environmental variable to a suitable location:
+
+    # you might want to add this to your .bashrc
+    $ export RUSTY_V8_MIRROR=$HOME/.cache/rusty_v8
```

Comment:
The value of `$HOME` is not a `file://` URL - which would make the `RUSTY_V8_MIRROR` an invalid endpoint - might want to prepend `file://` manually and modify the script that populates the cache to get rid of the `file://` protocol respectively.
```suggestion
    $ export RUSTY_V8_MIRROR=file://$HOME/.cache/rusty_v8
```

Reply:
Good catch. Thinking about it more, I think I'll make RUSTY_V8_MIRROR interpret it as a path unless it starts with `http:` or `https:`. Seems easier.